main.cc

```diff
@@ -1121,12 +1113,6 @@ int main(int ac, char** av) {
             api::set_server_done(ctx).get();
             supervisor::notify("serving");
             // Register at_exit last, so that storage_service::drain_on_shutdown will be called first
-            auto stop_redis_query_processor = defer([&redis_query_processor, cfg] {
```

Comment:
How is this destruction code not needed any more?

Reply:
The shutdown hook for redis is added now.